exporter/loadbalancingexporter/README.md

```diff
@@ -66,6 +66,12 @@ exporters:
         - backend-2:4317
         - backend-3:4317
         - backend-4:4317
+      ## use k8s service resolver, if collector runs in kubernetes environment
```

Comment:
This new resolver deserves more information than that. There are other parts of the documentation that needs to be changed, especially line 31.

Reply:
done